tests/test_cases.md

```diff
@@ -36,11 +36,14 @@ mozdownload --url=https://mozqa.com/index.html -d url-downloads
 mozdownload --url=https://mozqa.com/index.html --username=username --password=password
```

Comment:
Not directly related but we can remove all of those tests. Those have been automated now.